build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -20,9 +20,13 @@ import 'exceptions.dart';
 import 'options.dart';
 import 'phase.dart';
 
+BuildResults runWatch(
```

Comment:
nit: Lets add some doc comments (what is `until`?)

Reply:
Done